irods/manager/data_object_manager.py

```diff
@@ -460,6 +460,29 @@ def make_FileOpenRequest(**extra_opts):
             ret_value.seek(0,io.SEEK_END)
         return ret_value
 
+    def replica_truncate(self, path, desired_size, **options):
```

Comment:
Are there any conventions around naming functions?

I've noticed that some function names are closer to their C counterparts and others not so much.

So, why call this `replica_truncate` vs `truncate`?

Reply:
there's already a data_objects.truncate( ) , yes, although it's based on the existing (but deprecated I think?) truncate API

Reply:
Ah ok. Is the `truncate` function in PRC marked as deprecated? If not, please capture that in an issue.

As for this conversation, I think we're good. Resolving.

Reply:
Will do, the deprecation could happen in PRC v2.1.1  I guess but with a  3.0.0 milestone for implementing that goal, because PRC will still have to work with older servers and applications.

Reply:
Sounds good to me.

Reply:
Why not 2.1.0?

Reply:
Either is fine with me. Deprecate in 2.x, remove in 3.x.

Reply:
> Why not 2.1.0?

Sure.  `2.1.0` still has a few PR's to go, to be complete, and deprecating there would be fine too assuming we're in no hurry.